### PR TITLE
ARSSTB-575 Labels or legends for various pages + bugfixes

### PR DESCRIPTION
**Pull request by Ewan-James-Donovan**
### Type of change
- [ ] Breaking change
- [x] Non-breaking change
- [ ] Cosmetic change

### Description
Jira link: [Jira title](https://jira.tools.tax.service.gov.uk/browse/ARSSTB-575)

Please refer to the jira ticket. Please don't merge as it may be cumbersome to make adjustments after that.

### CODE REVIEW

File: conf/messages.cy [hunk 1]
```diff
@@ -291,14 +291,14 @@ describeTheConditions.paragraph.1 = Bydd rhoi disgrifiad manwl o’r cysylltiad
...
-whyIdenticalGoods.title = Esboniwch pam nad ydych wedi dewis Dull 1 i brisio’ch nwyddau
-whyIdenticalGoods.heading = Esboniwch pam nad ydych wedi dewis Dull 1 i brisio’ch nwyddau
-whyIdenticalGoods.label = Esboniwch pam nad ydych wedi dewis Dull 1 i brisio’ch nwyddau
-whyIdenticalGoods.paragraph.1 = Cyn i chi roi cynnig ar Ddull 2 dylech fod wedi defnyddio’r canlynol yn gyntaf:
+whyIdenticalGoods.title = Esboniwch pam nad ydych wedi defnyddio Dull 1
+whyIdenticalGoods.heading = Esboniwch pam nad ydych wedi defnyddio Dull 1
+whyIdenticalGoods.label = A allwch esbonio pam na allwch ddefnyddio Dull 1 i gyfrifo gwerth tollau eich nwyddau?
+whyIdenticalGoods.paragraph.1 = Cyn i chi roi cynnig ar Ddull 2, mae’n rhaid i chi fod wedi defnyddio Dull 1 (gwerth trafodiad y nwyddau a fewnforiwyd).
 whyIdenticalGoods.bulletPoint.1 = Dull 1 (gwerth trafodiad y nwyddau a fewnforiwyd)
 whyIdenticalGoods.paragraph.2 = Esboniwch pam rydych o’r farn nad yw Dull 1 yn addas i gyfrifo gwerth tollau eich nwyddau.
```
Comment (aaocs) [inline conf/messages.cy:299]: Remove these two keys if they are no longer used (they have been removed from messages.en)

**Review by aaocs — CHANGES_REQUESTED** [review]
2 keys to remove from messages.cy, and one question.

File: app/views/ExplainWhyYouHaveNotSelectedMethodOneToFiveView.scala.html [hunk 2]
```diff
@@ -58,15 +60,11 @@
...
-        @paragraph(Html(messages("explainWhyYouHaveNotSelectedMethodOneToFive.paragraph.2")))
 
         @hmrcCharacterCount(
             CharacterCountViewModel(
                 field = form("value"),
-                label = Label(
-                    content = Text(messages("explainWhyYouHaveNotSelectedMethodOneToFive.label")),
-                    classes = "govuk-visually-hidden",
-                )
+                label = LabelViewModel(Text(messages("explainWhyYouHaveNotSelectedMethodOneToFive.label"))).asLegendHeading()
```
[the remark this answers is not in the document]
Comment (Ewan-James-Donovan) [reply app/views/ExplainWhyYouHaveNotSelectedMethodOneToFiveView.scala.html:67]: This is 100% intentional, the labels are displayed as the questions, as per gov design system.

[the remark this answers is not in the document]
Comment (Ewan-James-Donovan) [reply app/views/ExplainWhyYouHaveNotSelectedMethodOneToFiveView.scala.html:67]: https://design-system.service.gov.uk/get-started/labels-legends-headings/

File: conf/messages.cy (hunk 1 above)
Comment (Ewan-James-Donovan) [reply conf/messages.cy]: Done

**Review by aaocs — DISMISSED** [review]

**Review by aaocs — DISMISSED** [review]

**Review by aaocs — APPROVED** [review]

**Pull request merged**